src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -659,4 +663,58 @@ public void clickOnCreateButton () {
   public void verifyInvalidConnectionErrorMessage(String errorMessageLocation) {
     CdfConnectionActions.verifyConnectionErrorMessage(errorMessageLocation);
   }
+
+  @Given("Open DataFusion Project with replication to configure pipeline")
+  public void openDataFusionProjectWithReplicationToConfigurePipeline() throws IOException, InterruptedException {
+    openCdf();
+    SeleniumDriver.getDriver().get(SeleniumDriver.getDriver().getCurrentUrl().replace(
+    SeleniumHelper.readParameters(ConstantsUtil.CDFURL), SeleniumHelper.readParameters(ConstantsUtil.REPLICATION_URL)));
+  }
+
+  @When("Enter input plugin property: {string} with pipelineName")
+  public void enterInputPluginPropertyWithPipelineName(String pluginProperty) {
+    WebElement pluginPropertyInput = PluginPropertyUtils.getInputPluginPropertyElement(pluginProperty);
+    String pipelineName = "TestPipeline-" + RandomStringUtils.randomAlphanumeric(10);
+    ElementHelper.sendKeys(pluginPropertyInput, pipelineName);
+  }
+
+  @And("Click on the button {string}")
+  public void clickOnTheButton(String button) throws InterruptedException {
+    CdfPluginPropertiesActions.clickOnPageButton(button);
+  }
+
+  @When("Select Source plugin: {string} from the replication plugins list")
+  public void selectSourcePluginFromTheReplicationPluginsList(String pluginName) {
+    CdfPluginPropertiesActions.selectReplicationSourcePlugin(pluginName);
+  }
+
+  @Then("Deploy the replication pipeline")
+  public void deployTheReplicationPipeline() {
+    CdfPluginPropertiesActions.deployReplicationPipeline();
+  }
+
+  @Then("Run the replication Pipeline")
+  public void runTheReplicationPipeline() {
+    CdfPluginPropertiesActions.runTheReplicationPipeline();
+  }
+
+  @Then("Open the Advanced logs")
+  public void openTheAdvancedLogs() {
+    CdfPluginPropertiesActions.openAdvancedLogs();
+  }
+
+  @Then("Close the replication pipeline logs and stop the pipeline")
+  public void closeTheReplicationPipelineLogsAndStopThePipeline() {
+    CdfPluginPropertiesActions.closeTheReplicationPipelineLogsAndStopThePipeline();
+  }
+
+  @Then("Verify that the Plugin is displaying an error message: {string}")
+  public void verifyThatThePluginIsDisplayingAnErrorMessage(String errorMessage) {
+    CdfPluginPropertiesActions.verifyErrorMessage(errorMessage);
+  }
+
+  @Then("Wait till the Review Assessment page is loaded in replication")
+  public void waitTillTheReviewAssessmentPageIsLoadedInReplication() {
+    CdfPluginPropertiesActions.waitTillTheReviewAssessmentPageLoaded();
```

Comment:
Instead of creating a step for page load, do you think we can use waits with the next button to wait for Assessment/Page load to be done. (Read - Implicit and Explicit waits)

Reply:
It is used in click on next step, but here only at this stage review assessment page takes time to load and this step is actually needed before clicking on next button again to proceed to next page.